### ISSUE TITLE
improve [simple_events_api_app]

### ISSUE THREAD

**Issue by Saif-Shines** [freshworks/marketplace-sample-apps#191]
_Maintainer's thoughts:_

This issue is ideally for pure beginners learning to make PRs and get familiar with open source contributions. Involve yourself to explore how apps on Freshworks developer platform run. You may not end up develop apps for Freshworks but experiece remains helpful where so ever you choose to build apps. Chrome Store? Web Apps? Atlassian apps? Using github CLI? Stripe CLI?

- [ ] Find sample code for `simple_events_api_app` app in the repository and tell us(in comments) if you were able to find it under 30s.
- [ ] (Optional) Add [summary in JSDoc commenting standards](https://jsdoc.app/tags-summary.html) and [elaborate parameters](https://jsdoc.app/tags-param.html) if you understand the code.
- [ ] Lower case the contents inside the `screenshots` folder (and `Screenshots` folder itself). If there's no screenshots just add it if it's a frontend app or else move on to next task. 
- [ ] Update the `README.md` file so that it follows [the standard format](https://github.com/freshworks/marketplace-sample-apps/blob/master/.github/PULL_REQUEST_TEMPLATE/APP_README_TEMPLATE.md). Don't worry about filling in all the details. Just give it a shot.
- [ ] Update the `README.md` file of the same app so that images inside of `screenshots` folder are displayed on the app README.md file.
- [ ] If you find any `iparams_test_data.json` inside of `config/` directory. Delete it.
- [ ] Re organise all the files and folders in following structure. Make sure the `html`, `css` and `js` files have right references to `src`s.
```
    .
    ├── README.md                   A file for future self and friends to learn about app
    ├── app                         A folder to place files needed for app's frontend components
    │   ├── index.html              The Entry point for the user to use the app
    │   ├── scripts                 JavaScript to handle frontend components business logic
    │   │   └── app.js
    │   ├── styles                  A folder to place all the styles for the app
    │   │   ├── images              A folder for all the images
    │   │   │   └── icon.svg
    │   │   └── styles.css
    │   └── views                   A folder to place all the pages
    │       └── modal.html
    ├── config                      A folder to place configuration pages
    │   └── iparams.json            A JSON oriented way to construct a configuration page
    └── manifest.json               A JSON file holding meta data for app to run on platform

```
- [ ] Install [Freshworks CLI](https://community.developers.freshworks.com/t/what-are-the-prerequisites-to-install-the-freshworks-cli/234)
- [ ] Rewrite UI elements using [Crayons kit](https://crayons.freshworks.com/components/). For example, this app uses buttons built using css(either from script tag or css file). Why not make it simple with crayons?
- [ ] Locally run the app by getting a trial account for relevant Freshworks' product (*Hint*: See `manifest.json`). We will let you figure out how to do that by exploring the [developer documentation](https://developers.freshworks.com/). How much time did you take to figure out? 2 min?
- [ ] Take a look at Events API section of relevant API developer documentation
- [ ] Facing difficulties? Reference to this issue and create a topic in [developer community](https://community.developers.freshworks.com/)

**Comment by jms429** [comment]
I think I would like to do this.

**Comment by jms429** [comment]
Yes, I was able to find simple_events_api_app sample code, assuming you know to look under Basic-Features to begin with. I think this answers your question.

**Comment by Saif-Shines** [comment]
Awesome @jms429 ! I have assigned this issue to you. See you soon raising a pull request.

**Comment by ravirajsubramanian** [comment]
Closed in #228
